src/generator.cpp

```diff
@@ -198,6 +198,7 @@ AnalyserVariablePtr Generator::GeneratorImpl::analyserVariable(const VariablePtr
 double Generator::GeneratorImpl::scalingFactor(const VariablePtr &variable) const
 {
     // Return the scaling factor for the given variable.
+
```

Comment:
I would just delete this the comment adds nothing here this method is self documenting.

Reply:
Done in commit 2fc440e6f.